src/View/View.php

```diff
@@ -168,7 +168,7 @@ class View implements EventDispatcherInterface
      *
      * @var string
      */
-    protected $_ext = '.ctp';
+    protected $_ext = '.php';
```

Comment:
Would be good if we could provide a tool to make this kind of mechanical change easier to do.

Reply:
Could just document this 3 liner needed (probably someone can improve it) to move the files and rename :)
```sh
git mv src/Template templates ;
find plugins -name 'Template' -exec sh -c 'git mv "$0" "$0"/../../templates' {} \; 2> /dev/null ;
find plugins templates -name '*.ctp' -exec sh -c 'git mv "$0" "${0%.ctp}.php"' {} \;

```